d3ploy/demand_driven_deployment_inst.py

```diff
@@ -286,7 +279,7 @@ def calc_diff(self, commod, time):
         if time not in self.commodity_supply[commod]:
             self.commodity_supply[commod][time] = 0.0
         supply = self.predict_supply(commod)
-        demand = self.predict_demand(commod, time)
+        demand = self.predict_demand(commod, time) * self.demand_std_dev
```

Comment:
I agree with roberto that it should be `1+self.demand_std_dev/100` or `1+self.demand_std_dev` depending on whether or not we tell the user to input a percentage

Reply:
But -- be careful with division by integers. Python 2.7 can, for edge cases, behave badly with std_dev/100... divide by 100.0 for robustness.